src/appendices/feedback.rst

```diff
@@ -0,0 +1,82 @@
+Feedback
+========
+
+Welcome LaunchCoder! We're so glad you're here. LaunchCoders help make 
+our curriculum stronger. You're a part of this community as a learner 
+and we want you to contribute to making our classes stronger.
+
+Questions
+---------
+
+Got a question? That's great! Questions mean you're learning. 
+Check out our `discourse forum <help.launchcode.org>`__. 
+
+Use the *Sign Up* button to create an account and get started 
+asking questions. Before you do, you'll want to read 
+`this helpful post <https://help.launchcode.org/t/how-to-ask-for-help/4756/2>`__
```

Comment:
This link directed me to the second part of the page and I had to scroll up in order to see the post. I almost thought that I went to a wrong page. I think removing the "/2" at the end of the link would direct one to the top of the page, which could solve the problem.